008-infinispan-getting-started/src/test/java/org/acme/quickstart/CacheResource.java

```diff
@@ -0,0 +1,39 @@
+package org.acme.quickstart;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Map;
+
+public class CacheResource implements QuarkusTestResourceLifecycleManager {
+
+    private static GenericContainer INFINISPAN = null;
+    private static final Integer INFINISPAN_PORT = 11222;
+
+    private static RemoteCacheManager cacheManager;
+
+    @Override
+    public Map<String, String> start() {
+
+        INFINISPAN =
+                new GenericContainer("jboss/infinispan-server:10.1.6.Final-2")
```

Comment:
Maybe it's a good idea to make this value configurable. Just in case we want to parameterise the project to be run using different infinispan versions.